submissions/opeyemiPeter/database/transportation-company.sql

```diff
@@ -19,24 +19,23 @@ SELECT * FROM registeredPassengers;
 -- Creating the Passenger trip details table
 CREATE TABLE tripDetails (
  id INT NOT NULL AUTO_INCREMENT,
- passengerId INT NOT NULL,
- passengerClass INT NOT NULL,
- passengerTicketNo VARCHAR(20) NOT NULL,
+ passengerId INT NOT NULL Default(1),
```

Comment:
This shouldn't have a default value. There is no need for that

Reply:
the CSV file couldn't import correctly until I added the default value.

Reply:
Hmmm, kindly specify the length of INT then. E,g INT(11)

Reply:
alright